superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -550,28 +550,13 @@ class Header extends React.PureComponent {
           {this.state.showingPropertiesModal && (
             <PropertiesModal
               dashboardId={dashboardInfo.id}
+              dashboardInfo={dashboardInfo}
```

Comment:
this is from: https://github.com/apache/superset/pull/17570/files it looks like it wasn't added correctly in a merge. I added it here to pass CI though I think it will be eliminated when the feature branch gets rebased and merged into master.